File: ABC127A.java
import java.util.*;

public class Main {
    public static void main(String[] args) throws Exception {
        // Your code here!
        Scanner sc = new Scanner(System.in);
        int age = sc.nextInt();
        int price = sc.nextInt();
        
        if (age >= 13) {
            System.out.println(price);
        } else if (age <= 5) {
            System.out.println(0);
        } else {
            System.out.println(price / 2);
        }
    }
}

File: C/ABC127A.c
#include <stdio.h>
int main(void){
    // Your code here!
    int age, price;
    scanf("%d%d", &age, &price);
    
    if (age >= 13) {
        printf("%d", price);
    } else if (age <= 5) {
        printf("0");
    } else {
        printf("%d", price / 2);
    }
    return 0; 
}
